content/en/blog/_posts/2021-11-18-persistentvolume-reclaim-improvements.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: blog
+title: "Kubernetes 1.23 PersistentVolume reclaim improvements"
+date: 2021-11-18T10:00:00-08:00
+slug: kubernetes-1-23-persistentvolume-reclaim-improvements
+---
+
+**Author:** Deepak Kinni (VMware)
+
+[Persistent Volumes](/docs/concepts/storage/persistent-volumes/) (or PVs for short) are
+associated with [Reclaim Policy](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaim-policy).
+The Reclaim Policy is used to determine the actions that need to be taken by the storage
+backend on deletion of the PV.
+Where the reclaim policy is `Delete`, the expectation is that the storage backend
+releases the storage resource that was allocated for the PV. In essence, the reclaim
+policy needs to honored on PV deletion.
+
+With the recent Kubernetes v1.23 release, an alpha feature lets you configure your
+cluster to behave that way and honor the configured reclaim policy.
+
+
+## How did reclaim work in previous Kubernetes releases?
+
+[Persistent Volume Claim](/docs/concepts/storage/persistent-volumes/#Introduction) (or PVC for short) is
```

Comment:
This should be PersistentVolumeClaim (no spaces) - see https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects